js/scripts.js

```diff
@@ -1,11 +1,9 @@
 function getTriangleArea(a, h) {
-    if ( (a > 0) && (h > 0) ) {
-        console.log('Prawidłowe dane');
+    if ( (a > 0) && (h > 0) ) { 
+        return a*h/2;
     } else {
```

Comment:
else in this case is unnecessary :) you can write `if` and when `if` contain `return`, `else` you can remove - keep only code inside `else`